prow/scripts/cluster-integration/kyma-integration-k3s.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+export KYMA_SOURCES_DIR="./kyma"
+export LOCAL_KYMA_DIR="./local-kyma"
+
+install::prereq() {
```

Comment:
This convention we use (or want to use) for the functions that are in the `lib` directory. Please change the functions names

Reply:
@Ressetkk ok, changed.